src/config.rs

```diff
@@ -425,29 +425,84 @@ pub struct HtmlConfig {
     pub livereload_url: Option<String>,
     /// Should section labels be rendered?
     pub no_section_label: bool,
+    /// Search settings.
+    pub search: Search,
 }
 
 /// Configuration for tweaking how the the HTML renderer handles the playpen.
 #[derive(Debug, Clone, PartialEq, Serialize, Deserialize)]
 #[serde(default, rename_all = "kebab-case")]
 pub struct Playpen {
-    /// The path to the editor to use. Defaults to the [Ace Editor].
-    /// 
+    /// The path to the editor to use. If not specified, will use the [Ace Editor] implementation
+    /// included in MDBook. Default: `""`.
+    ///
     /// [Ace Editor]: https://ace.c9.io/
     pub editor: PathBuf,
-    /// Should playpen snippets be editable? Defaults to `false`.
+    /// Should playpen snippets be editable? Default: `false`.
     pub editable: bool,
 }
 
 impl Default for Playpen {
     fn default() -> Playpen {
         Playpen {
-            editor: PathBuf::from("ace"),
+            editor: PathBuf::new(),
             editable: false,
         }
     }
 }
 
+/// Configuration of the search functionality of the HTML renderer.
+#[derive(Debug, Clone, PartialEq, Serialize, Deserialize)]
+#[serde(default, rename_all = "kebab-case")]
+pub struct Search {
+    /// Enable in browser searching. Default: `true` (if `search` feature is enabled).
+    pub enable: bool,
```

Comment:
Is it necessary to have an `enabled` flag? I would have thought we can see search is enabled because a `output.html.search` is either present or not.

So an alternative to the `enabled` flag would be to change the `search` field in `HtmlConfig` to be `Option<Search>`... Thoughts?

Reply:
Agreed. Especially now that the JS is conditionally included, the flag is not very useful.

Reply:
Hm, that would change search to be disabled by default. Is that your intention?

Reply:
I dunno, to be honest it's one of those features you'll always want for your document. 

It should probably be always turned on unless you manually deactivate the `search` feature flag when installing `mdbook` (e.g. when packaging for Debian).